src/recipe/Baked.Recipe.Service.Application/Localization/AspNetCore/AspNetCoreLocalizationFeature.cs

```diff
@@ -14,6 +15,51 @@ public class AspNetCoreLocalizationFeature(CultureInfo _language,
 {
     public void Configure(LayerConfigurator configurator)
     {
+        configurator.ConfigureGeneratedFileCollection(files =>
+        {
+            if (configurator.IsNfr())
+            {
+                return;
+            }
+
+            var localeDir = Path.Combine(Assembly.GetEntryAssembly()?.Location ?? throw new("'EntryAssembly' shoul have existed"), "../../../../Locales");
+
+            configurator.UsingLocaleDictionary(locales =>
```

Comment:
this doesn't read nice. need to be more strict on what locale is. locale seems like a set of key and value for a specific language. locale dictionary then means a dictionary of locales? but this case it means something like locale key collection.

we may say this is a locale template? to indicate that this is a base for any locale.

```csharp
configurator.UsingLocaleTemplate(localeTemplate =>
{
  ...
});
```